### PR TITLE
release v4.19.0.1

### PR DESCRIPTION
**Pull request by sulewskiprzemyslaw**
Signed-off-by: Przemyslaw Sulewski <przemyslaw.sulewski@3mdeb.com>

### CODE REVIEW

File: _posts/2023-01-27-PC-Engines_Firmware-v4-19-0-1.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Mainline:
 
 ### Changed
 
-- **Rebased** with official coreboot repository **commit** 2ccbcc5
+- **Rebased** with official coreboot repository **commit** 8d3e714
```
Comment (miczyg1) [inline _posts/2023-01-27-PC-Engines_Firmware-v4-19-0-1.md:15]: 2ccbcc5 is the right commit hash

File: index.html [hunk 2]
```diff
@@ -416,7 +416,7 @@ <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.17
         <ul>
             <li>Changed:</li>
             <ol>
-              <li>Rebased with official coreboot repository commit 2ccbcc5</li>
+              <li>Rebased with official coreboot repository commit 8d3e714</li>
```
Comment (miczyg1) [inline index.html:419]: 2ccbcc5 is the right commit hash

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: _posts/2023-01-27-PC-Engines_Firmware-v4-19-0-1.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+# PC Engines apu coreboot Open Source Firmware v4.19.0.1
+
+## Key changes
+
+Mainline:
+
+### Changed
+
+- **Rebased** with official coreboot repository **commit** 8d3e714
+- Removed configuration and mainboard files for apu1
```
Comment (miczyg1) [inline _posts/2023-01-27-PC-Engines_Firmware-v4-19-0-1.md:16]: PLease use the bulletpoint from coreboot's CHANGELOG.md file https://github.com/pcengines/coreboot/blob/release/CHANGELOG.md?plain=1#L19

File: index.html [hunk 4]
```diff
@@ -399,12 +400,80 @@ <h1>Documentation</h1>
...
+        <div class="wrapper">
+          <!--<div class="break"></div>insert this code when a new update comes out to separate a new version from the current.-->
+          <div class="smaller-margin" id="mr-63">
+            <h2 class="text-middle">Mainline releases</h2>
+            <!-- h2 tag needs to be moved to the newest version -->
+          <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.17.0.3...rel_4.19.0.1">v4.19.0.1</a></h4>
+        </div>
+  
+        <div class="dimmed-text">
+          <p>Release date: 2023-01-27</p>
```
Comment (miczyg1) [inline index.html:413]: Date update? tag was placed on 31th of January

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: _posts/2023-01-27-PC-Engines_Firmware-v4-19-0-1.md (hunk 1 above)
Comment (sulewskiprzemyslaw) [reply _posts/2023-01-27-PC-Engines_Firmware-v4-19-0-1.md:15]: fixed

File: _posts/2023-01-27-PC-Engines_Firmware-v4-19-0-1.md [hunk 5]
```diff
@@ -25,7 +25,7 @@ point of given release specified in CHANGELOG (CHANGELOG.md, gitlab-ci.yml
 and regression.sh excluded from statistics). Check the statistics with:
 
 ```
-git diff --stat 2ccbcc5 ':(exclude).gitlab-ci.yml' ':(exclude)CHANGELOG.md' ':(exclude).gitlab-ci/regression.sh'
+git diff --stat 8d3e714 ':(exclude).gitlab-ci.yml' ':(exclude)CHANGELOG.md' ':(exclude).gitlab-ci/regression.sh'
```
[the remark this answers is not in the document]
Comment (sulewskiprzemyslaw) [reply _posts/2023-01-27-PC-Engines_Firmware-v4-19-0-1.md:28]: fixed

File: index.html (hunk 2 above)
Comment (sulewskiprzemyslaw) [reply index.html:419]: fixed

File: _posts/2023-01-27-PC-Engines_Firmware-v4-19-0-1.md (hunk 3 above)
Comment (sulewskiprzemyslaw) [reply _posts/2023-01-27-PC-Engines_Firmware-v4-19-0-1.md:16]: improved

File: index.html (hunk 4 above)
Comment (sulewskiprzemyslaw) [reply index.html:413]: Could we use in this place tag placed date or release date?

Comment (miczyg1) [reply index.html]: Based on the previous version's dates in this file, it should be the date of release publication on github.io page (so today 2023-02-02)

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**